galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -3,14 +3,16 @@
 from django.db import transaction
 
 from rest_framework.exceptions import ValidationError
-from rest_framework.serializers import ModelSerializer, SlugRelatedField
+from rest_framework.serializers import ModelSerializer, SlugRelatedField, URLField
 
 from galaxy_api.api import models
 from galaxy_api.auth import models as auth_models
 from galaxy_api.auth import auth
 
 
 class NamespaceLinkSerializer(ModelSerializer):
+    url = URLField()
```

Comment:
Wouldn't it be appropriate for the model to use a model `URLField` in the first place? https://docs.djangoproject.com/en/3.0/ref/models/fields/#urlfield